### ISSUE TITLE
Python Vispy fails to build - checksum

### ISSUE THREAD

**Issue by jonpatterns** [acxz/pkgbuilds#232]
I get an error building python-vispy from AUR - error message to do with checksum:

```
==> Retrieving sources...
  -> Found python-vispy-0.12.2.tar.gz
==> Validating source files with sha256sums...
    python-vispy-0.12.2.tar.gz ... FAILED
==> ERROR: One or more files did not pass the validity check!
Failed to build python-vispy

```
I get a different checksum for python-vispy-0.12.2.tar.gz

`0e23eacb8adb0fd37125f1b58e9a1e21ca31f0318793c54d20ce99a7d338273f  python-vispy-0.12.2.tar.gz`

In pkgbuild checksum is 

`73e7c081d01fc3f7e1693d54d265239abc2afdcefd927449f2ef0a74ee9c49d0`

**Comment by acxz** [comment]
Nice find!
Do you mind submitting a PR for this?

**Comment by jonpatterns** [comment]
Done.

**Comment by acxz** [comment]
closed with ee58873
